boards/stm32f4discovery/stm32f4discovery.config

```diff
@@ -0,0 +1,3 @@
+CONFIG_MODULE_USBUS=y
```

Comment:
IMHO, this will cause a problem when compiling `tinyUSB` apps.

Reply:
Hm I just copied that from `stm32f429i-disco`, not sure how this could be done better in Kconfig.

Reply:
```
TEST_KCONFIG=1 BOARD=stm32f4discovery make -C tests/pkg_tinyusb_cdc_msc info-modules
```
```
=== [ATTENTION] Testing Kconfig dependency modelling ===
[genconfig.py]:ERROR-=> The choice MODULE_STDIO_CDC_ACM (defined at /home/gs/src/RIOT-Xtensa-ESP.esp-idf-4.4/sys/usb/usbus/cdc/acm/Kconfig:57) was selected but was not set.

```

Reply:
I have also problems with `stm32f429i-disco`. Maybe they are not compiled with `TEST_KCONFIG=1` in CI.

Reply:
That's the reason why enabling the `tinyusb_device` feature for boards with a single USB interface doesn't work and why PR #18998 which tried to enable `tinyusb_device` for all such boards hangs since 3 months.

Reply:
Probably the problem in Kconfig could be solved for the moment as in `boards/esp32s3-pros3/Kconfig` by overriding the default settings in `STDIO_IMPLEMENTATION`.

Reply:
```
choice STDIO_IMPLEMENTATION
    bool "STDIO implementation"
    depends on TEST_KCONFIG
    default MODULE_STDIO_CDC_ACM if MODULE_USBUS
    default MODULE_STDIO_TINYUSB_CDC_ACM if MODULE_TINYUSB_DEVICE
endchoice
```